frontend/src/basiclands.js

```diff
@@ -0,0 +1,176 @@
+/* uncomment this and run this file to output lands! */
```

Comment:
Should this be kept in the code ? It seems just a momentaneous helper. But this is fine if it's needed.

Reply:
I want future devs to know its easy to change the default lands included to a different set - eg m22

Reply:
Have left a more clear comment with instructions here